admin/src/pages/HomePage/components/ControllableCombobox/index.tsx

```diff
@@ -16,6 +16,14 @@ type ControllableComboboxProps = {
   disabled: boolean;
 };
 
+function findTextValue(options: ControllableComboboxProps['options'], value: ControllableComboboxProps['value']): string {
+  if (!value) {
+    return '';
+  }
+
+  return options.find(o => o.value === value)?.label ?? '';
+}
+
```

Comment:
@jorrit may I ask you to change it to `const findTextValue ...` for consistency?

Reply:
Of course, I've modified the commit.